hs_core/models.py

```diff
@@ -761,9 +765,9 @@ def validate_identifiers(cls, identifiers):
             for id_name in cls.supported_identifiers:
                 id_link = identifiers.get(id_name, '')
                 if id_link:
-                    if not id_link.startswith(cls.supported_identifiers[id_name]) \
-                            or len(id_link) <= len(cls.supported_identifiers[id_name]):
-                        raise ValidationError("URL for {} is invalid".format(id_name))
+                    regex = cls.supported_identifiers[id_name]
+                    if not re.match(regex, id_link):
```

Comment:
We don't have a regex for `ResearcherID`, so I was expecting this would raise an exception. But some reason the test is passing though.

Reply:
Sorry I was reading the code wrong.

Reply:
no worries, thanks for checking!